src/ensembl/README.md

```diff
@@ -1,38 +1,46 @@
-# Ensembl 2020 Frontend
+# Running the ensembl-client application
 
-## Setup
+## Installation
 
-The tools that need setting up are Node.js and the IDE. To setup Node.js you only need to run `npm install`. This will install all the necessary NPM packages for app development and the build.
+The Ensembl client runs on NodeJS, and all supplementary libraries required for the genome browser (e.g. Rust) are included in this repository. To setup Node.js you only need to run `npm install`. This will install all the necessary NPM packages for app development and the build.
 
-### Build Scripts
+## Build Scripts
 
-There are several script commands that have been baked into the NPM configuration. They have been briefly described below:
+There are several script commands that have been baked into the NPM configuration. They are briefly described below:
```

Comment:
> have been baked into the NPM configuration

That's an unfortunate turn of phrase.
What we are trying to say here is that there are some scripts for building or developing the project, run with the npm command. But I am not even sure we need this introductory paragraph — anyone interested will likely know what npm scripts are. Compare with the [Readme](https://github.com/facebook/create-react-app) of Create React App — despite it being a very beginner-friendly project, it just plunges into npm and npx commands without any introduction.

Reply:
Paragraph deleted.